test/unit/model/test_confidence_interval_applicator.py

```diff
@@ -11,20 +12,86 @@
 
 class MockModel:
     confidence_interval = pd.DataFrame()
-
-    def predict(self, input, quantile):
+    standard_deviation = pd.DataFrame(
+        {
+            "stdev": [
+                1.1089453301,
+                2.9508361224,
+                1.6705985526,
+                1.9371298645,
+                3.2265538352,
+            ],
+            "hour": [0, 1, 2, 3, 4],
+            "horizon": [47, 47, 47, 47, 47],
+        }
+    )
```

Comment:
I don't see how this is needed in the unit test. A small docstring on why this is required would help the next person who looks at it

Reply:
Yes good suggestion. Added a small docstring.